server/index.js

```diff
@@ -214,4 +223,48 @@ api.get('/torrents/:infoHash/archive', findTorrent, function (req, res) {
   archive.finalize();
 });
 
+function parseDom(body) {
+  var parseHandler = new htmlparser.DefaultHandler(function (err, dom) {
+    if (err) {
+      console.error("Error: " + err);
+    }
+  });
+  var parser = new htmlparser.Parser(parseHandler);
+  parser.parseComplete(body);
+  return parseHandler.dom;
+}
+
+api.get('/search/:param', function (req, res) {
+  request(`https://1337x.to/search/${encodeURIComponent(req.params.param)}/1/`, function(error, response, body) {
```

Comment:
I've intentionally avoided adding any trackers or search engines in order to avoid piracy claims and DMCA requests. If you would like, you could fork this project and maintain this feature yourself.